dtfs-central-api/src/services/utilisation-report-data-validator/utilisation-report-cell-validators/generate-payment-exchange-rate-error.test.ts

```diff
@@ -32,33 +32,33 @@ describe('generatePaymentExchangeRateError', () => {
     expect(exchangeRateError).toEqual(null);
   });
 
-  it('returns an error if payment currency is null and payment exchange rate is not a number', async () => {
+  it('returns an error if payment currency is null and payment exchange rate is not a number', () => {
     const csvDataRow = {
       exporter: {
         value: testExporterName,
-        column: 1,
+        column: 'A',
         row: 1,
       },
       'fees paid to ukef currency': {
         value: 'GBP',
-        column: 2,
+        column: 'B',
         row: 1,
       },
       'payment currency': {
         value: null,
-        column: 2,
+        column: 'C',
         row: 1,
       },
       'payment exchange rate': {
         value: 'abc',
         row: 1,
-        column: 3,
+        column: 'D',
```

Comment:
I think in the majority of cases the fees paid to ukef currency is highly relevant to the test case, so I'd actually be uncomfortable pulling it out. Appreciate the importance of being DRY, but I think with the data row arrays refactors are likely to make the tests much more brittle for not much benefit.

I'm not sure I understand the value of pulling out the exporter name?

If we were wanting to truly avoid any repetition at all we would define `MOCK_NON_EMPTY_STRING = 's'` and then each test becomes:

```
const expectedError = {
      errorMessage: 'Base currency must have an entry',
      column: 'A',
      row: 1,
      value: null,
      exporter: MOCK_NON_EMPTY_STRING,
    };

    const baseCurrencyError = generateBaseCurrencyError(nullBaseCurrency, MOCK_NON_EMPTY_STRING);
```
But I personally think that is harder to read and follow?

Reply:
@RLavender98 I mean the overall CSV row mocks, not just the exporter name. E.g if you search for this: `column: 'D', looks like there's a lot of repitition and we could create a common fixture for mock CSV rows?

E.g in here `generate-payment-currency-error.test`, the same mock row is duplicated 3x places with only 1/2 very small differences

Could we create e.g `mockCsvDataRow`? Looks like it could remove a fair amount of LOC 🚀

Reply:
Thanks @RLavender98 , I think it would be worth doing, but not a blocker for this PR since they're existing mocks and this PR only updates one value (column) in these mocks.

Even with test specifics, it looks like we could reduce a lot of LOC for example a test could e.g:

```
csvDataRow['payment exchange rate'].value = null;
```

And it'd save ~22 lines of mocks, for 1x test. Multiplying this for all the instances, we could be saving ~500 lines of mocks in total